lucille-core/src/main/java/com/kmwllc/lucille/util/ThreadNameUtils.java

```diff
@@ -0,0 +1,12 @@
+package com.kmwllc.lucille.util;
+
+public class ThreadNameUtils {
+  public static final String THREAD_NAME_PREFIX = "Lucille";
+
+
+  public static String setThreadPrefix(String name) {
```

Comment:
methods that start with "set" are usually setters that update the state of the object you're calling them on
in this case, we have a static method that doesn't set or update any internal state
suggest changing this to something like createName() or getThreadName()

Reply:
changed to createName in [cdc333a](https://github.com/kmwtechnology/lucille/pull/172/commits/cdc333a75b671f998894dc5e830c75e13bb5819d)